ddev/pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
 ]
 dependencies = [
-    "datadog-checks-dev[cli]~=18.0",
```

Comment:
Does this need to be in its own PR? I think historically this isn't in a release PR?